### DISCUSSION TITLE
Usage of fs

### DISCUSSION THREAD

**Discussion by anli-xsigns** [quasarframework/quasar#9494] [CLI - PWA mode] [answered]
I want to use fs module to read some directory. However compile fails with

```
 App •  ERROR  •  UI  in ./src/router/index.ts

Module not found: Can't resolve imported dependency "fs"
Did you forget to install it? You can run: yarn add fs
```
What can I do to use fs? Some people suggest to set fs to empty in webpack config, but Razvan suggests to not do it (https://github.com/quasarframework/quasar-cli/issues/121).

**Comment by ontwikkelfabriek** [comment, marked as answer]
@anli-xsigns 
`fs` only runs on nodejs , not on the clients browser.(PWA, SPA)
